communication/mavlink_waypoint_handler.cpp

```diff
@@ -1685,15 +1674,15 @@ void Mavlink_waypoint_handler::dubin_state_machine(waypoint_local_struct_t* wayp
 // PUBLIC FUNCTIONS IMPLEMENTATION
 //------------------------------------------------------------------------------
 
-Mavlink_waypoint_handler::Mavlink_waypoint_handler(Position_estimation& position_estimation_, Navigation& navigation_, const ahrs_t& ahrs_, State& state_, const Manual_control& manual_control_, Mavlink_message_handler& message_handler, const Mavlink_stream& mavlink_stream_, conf_t config):
+Mavlink_waypoint_handler::Mavlink_waypoint_handler(INS& ins_, Navigation& navigation_, const ahrs_t& ahrs_, State& state_, const Manual_control& manual_control_, Mavlink_message_handler& message_handler, const Mavlink_stream& mavlink_stream_, conf_t config):
```

Comment:
I think we mess up in the constructor, underscore **only** for members not for parameters of the constructor
to my understanding, it should be something like that 
cstr(type& variable1, ... conf_t conf):
variable1**_**(variablea),
...,
conf**_**(conf)
{ }